src/runtime-rs/crates/hypervisor/src/device/util.rs

```diff
@@ -69,9 +69,55 @@ pub(crate) fn get_virt_drive_name(mut index: i32) -> Result<String> {
     Ok(String::from(PREFIX) + std::str::from_utf8(&disk_letters)?)
 }
 
+// do_increase_count
```

Comment:
Remove it or expand on this, as this comment doesn't provide any valuable information.